lib/Importer/ONNX.cpp

```diff
@@ -427,21 +437,47 @@ void ONNXModelLoader::loadNetwork(onnx::GraphProto &net) {
   /// Load the network operators:
   for (int i = 0; i < net.node_size(); i++) {
     auto &op = net.node(i);
-    loadOperator(op);
+    if (!loadOperator(op)) {
+      unexpectedNodeError(op, "Unsupported operator.");
+    }
   }
 
   assert(net.output_size() && "Network needs external outputs defined.");
```

Comment:
Can we move this code, so that `loadNetwork` could be used in both cases?

Reply:
fixed this